src/dsp_tools/commands/xmlupload/models/ontology_diagnose_models.py

```diff
@@ -0,0 +1,102 @@
+# sourcery skip: use-fstring-for-concatenation
+
+from dataclasses import dataclass, field
+from pathlib import Path
+
+import pandas as pd
+
+from dsp_tools.models.exceptions import UserError
+
+
+@dataclass(frozen=True)
+class Ontology:
+    """This class saves the properties and the classes from an ontology."""
+
+    classes: list[str] = field(default_factory=list)
+    properties: list[str] = field(default_factory=list)
+
+
+@dataclass
+class OntoCheckInformation:
+    """This class saves information that is needed to check if consistency with the ontology."""
+
+    default_ontology_prefix: str
+    onto_lookup: dict[str, Ontology]
+    save_location: Path
```

Comment:
This field seems counter-intuitive to me: What does a save location have to do with an ontology check information? Could you explain it in the docstring?